azure-pipelines.yml

```diff
@@ -1,15 +1,18 @@
-# Run generate-doc script.
+# Run generate-docs script.
 
 # TODO: push on master shimming-toolbox
-#trigger:
+trigger:
+- master
```

Comment:
This is triggering on PRs and on commits, which is bad: I want the website to only be rebuilt when master is merged.

Docs here https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#ci-triggers -- I don't know what I misread but I misread something.